src/pages/Donors/Donors.tsx

```diff
@@ -1,16 +1,63 @@
 import {
   IonButton,
   IonButtons,
+  IonContent,
   IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
   IonPage,
   IonTitle,
   IonToolbar,
 } from "@ionic/react";
+import {
+  Maybe,
+  Message,
+  useCustomerConversationsListQuery,
+} from "generated/graphql";
 import useAuth from "hooks/useAuth/useAuth";
 import React from "react";
 
+interface LatestMessagePreviewProps {
+  messages: Array<Maybe<{ __typename?: "Message" } & Pick<Message, "content">>>;
+}
+const LatestMessagePreview: React.FC<LatestMessagePreviewProps> = ({
```

Comment:
I recommend transforming that into `MessagePreview` (not _Latest_), and it just takes a single message as props.
It'll make it more useful/reusable to preview any kind of message (especially if it becomes a standalone component)